docs/design/alias-configuration.md

```diff
@@ -27,10 +27,26 @@ This document explains how additional Crypto transaction flows may be used to cr
 
 ### Alias to Public Key Validation
 
-### Support Public Key or Ethereum Address Alas
+- In case of ED25519 the provided alias must match the public key of the account that we want to create
+- In case of the ECDSA the alias must match the public key or be the public address of the ECDSA key provided/set on the account (when the alias is the public address of the ECDSA key and the key is not set on the account we have to create the account with empty public key field and to store the public address as alias)
+- We can use MiscUtils.isSerializedProtoKey logic to validate keys
+- Additionally we can validate with regex for evm_address (if hex encoded string), otherwise if bytes array check its length
+- We can check existing aliases using AliasManager.lookupIdBy method
```

Comment:
Note this may not extend long term.
Currently the Alias map includes public address but we should account for scenarios where it only contains public keys.

Reply:
Currently the aliases map contains entries for public keys and public addresses that are derived from public keys.

For the case of the creation of hollow account with public address (to be implemented) I can see that the aliases map will contain entry only for public address and not for the corresponding public key. 
Adding the public key entry corresponding to the public address will be done as finalizing the lazy account creation, which will leave the aliases map in the same state we currently have i.e. entries with public keys and public addresses corresponding to those public keys.

What is the case we may have in the future in which we can have only public key entries?